src/chunk_constraint.c

```diff
@@ -253,12 +260,17 @@ ts_chunk_constraints_add_from_tuple(ChunkConstraints *ccs, const TupleInfo *ti)
 		hypertable_constraint_name = DatumGetName(DirectFunctionCall1(namein, CStringGetDatum("")));
 	}
 
+	/* check if it's a correlated constraint */
+	if (strncmp(CC_DIM_PREFIX, NameStr(*constraint_name), strlen(CC_DIM_PREFIX)) == 0)
```

Comment:
Is it possible to have a better way of knowing that this is a correlated constraint apart from encoding it in the name?

Reply:
Consulting the dimension slice and the dimension catalog to get this information now.